ntoskrnl/ke/amd64/cpu.c

```diff
@@ -234,6 +234,61 @@ KiGetFeatureBits(VOID)
     return FeatureBits;
 }
 
+#if DBG
+VOID
+NTAPI
```

Comment:
As said in one previous comment, if function is internal and not a callback, NTAPI is superfluous.